src/scripts/clipperUI/components/ariaGroupingProps.ts

```diff
@@ -0,0 +1,4 @@
+export interface PropsForAriaGrouping {
```

Comment:
Call this ariaSetProps.ts? Sets in aria have a very defined connotation so we shouldn't use "set" and "group" interchangeably